{
  "comments": [
    {
      "key": {
        "uuid": "29cecbc8_fa4e3db6",
        "filename": "plugins/tempest/vars/tests/kuryr.yml",
        "patchSetId": 11
      },
      "lineNbr": 9,
      "author": {
        "id": 1001189
      },
      "writtenOn": "2020-06-08T08:02:05Z",
      "side": 0,
      "message": "is this intended or mistake?\ni guess it may be to enable usage of newer version which requires py3 venvs?",
      "revId": "501ee71f914ccbe55d7ab9529acb5a8a289a96ce",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}